SYCL/Basic/host-task-dependency.cpp

```diff
@@ -146,7 +155,7 @@ void test() {
   Context Ctx{{false}, Queue, {10}, {10}, {10}, {}, {}};
 
   // 0. setup: thread 1 T1: exec smth; thread 2 T2: waits; init flag F = false
-  auto A1 = std::async(std::launch::async, Thread1Fn, &Ctx);
+  auto A1 = std::async(std::launch::async, Thread1Fn<true>, &Ctx);
```

Comment:
Is it intended to be hardcoded and the next line is unspecified?

Reply:
I am not sure what the question is about. Only 'Thread1Fn' was changed to a template function. 'Thread2Fn' was not changed same way.

Reply:
Fixed in 87e9d5f3997444130bf159348b59093867032b53. Thank you!